schema/2.0.0/seurat_encoding.md

```diff
@@ -237,4 +237,64 @@ The matrix with reductions is stored in the `cell.embeddings` slot of `SeuratObj
 
 ### Dataset Metadata
 
-At the moment a Seurat object downloaded from cellxgene does not contain the dataset metadata stored in the AnnData slot `uns`.
+The dataset metadata from the AnnData slot `uns` is stored in the Seurat object in the slot `misc`. 
```

Comment:
This should follow a similar style to your previous text and not include references to AnnData.

Dataset metadata is stored in the `misc` slot. 

and

Only fields defined in the [schema](./schema.md/#uns-dataset-metadata) are transferred to the Seurat object

Reply:
Do we need/want to include a statement about why `uns` is being stored in `misc`? i.e.:
* https://app.zenhub.com/workspaces/single-cell-5e2a191dad828d52cc78b028/issues/chanzuckerberg/single-cell-curation/149#issuecomment-934641155